src/pages/Liquidated.tsx

```diff
@@ -7,22 +7,58 @@ import { LiquidatedVaults } from '@/widgets/LiquidatedVaults';
 import { LiquidatedVaultCountCard } from '@/widgets/LiquidatedVaultCountCard';
 import { VaultStatesChart } from '@/widgets/VaultStatesChart';
 import { populateMissingDays, subQueryFetcher } from '@/utils';
-import { LIQUIDATIONS_DASHBOARD, VAULT_STATE_DAILIES_QUERY } from '@/queries';
-import { GraphData, LiquidationDashboardResponse, VaultStateDailyResponse } from '@/types/liquidation-types';
+import { LIQUIDATED_VAULTS_NEXT_PAGES_QUERY, LIQUIDATIONS_DASHBOARD, VAULT_STATE_DAILIES_QUERY } from '@/queries';
+import {
+  GraphData,
+  LiquidationDashboardResponse,
+  VaultLiquidationsNode,
+  VaultStateDailyResponse,
+} from '@/types/liquidation-types';
 import { GRAPH_DAYS } from '@/constants';
 import { ErrorAlert } from '@/components/ErrorAlert';
 
 export function Liquidated() {
   const { data, isLoading, error } = useSWR<AxiosResponse, AxiosError>(LIQUIDATIONS_DASHBOARD, subQueryFetcher);
-  const response: LiquidationDashboardResponse = data?.data?.data;
+  const liquidatedVaultsDataResponse: LiquidationDashboardResponse = data?.data?.data;
 
-  const boardAuxes: { [key: string]: number } = response?.boardAuxes?.nodes?.reduce(
+  const totalVaultsCount = liquidatedVaultsDataResponse?.vaultLiquidations.totalCount || 1;
+  const pageCount = Math.ceil(totalVaultsCount / 100) - 1;
+  const {
+    data: liquidatedVaultsNextPages,
+    error: nextPagesError,
+    isLoading: nextPagesIsLoading,
+  } = useSWR<AxiosResponse, AxiosError>(
+    pageCount ? LIQUIDATED_VAULTS_NEXT_PAGES_QUERY(pageCount) : null,
+    subQueryFetcher,
+  );
+
+  const pageError = error || nextPagesError;
+
+  let liquidatedVaultsDataAppended: LiquidationDashboardResponse = liquidatedVaultsDataResponse;
+  if (liquidatedVaultsNextPages) {
+    const vaultPages: {
+      [key: string]: {
+        nodes: Array<VaultLiquidationsNode>;
+      };
+    } = liquidatedVaultsNextPages?.data?.data || {};
+    const nextVaults = Object.values(vaultPages).flatMap((openVaultsPage) => openVaultsPage.nodes);
```

Comment:
Should we move the type definition to the relevant types file?

Reply:
i think we can keep it here since its specific to this variable and wont be used anywhere else.